db/seeds.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+Greeting.create(message: 'Hello World!')
+Greeting.create(message: 'Hi there!')
+Greeting.create(message: 'Welcome!')
+Greeting.create(message: 'Greetings!')
+Greeting.create(message: 'Hola!')
```

Comment:
- [ ] Good job adding the seed data as requested by the previous reviewer, however, you have an error as `Greeting` is an uninitialized constant (use the `Message` model to add data instead) and the messages `table` does not have a `message` field but rather has a `content` field. Fix this issue by re-writing your code as follows:

```
Message.create(content: 'Hello World!')
Message.create(content: 'Hi there!')
Message.create(content: 'Welcome!')
Message.create(content: 'Greetings!')
Message.create(content: 'Hola!')
```